app/Http/Controllers/TaskController.php

```diff
@@ -48,22 +65,16 @@ public function create()
      * @param  \Illuminate\Http\Request  $request
      * @return \Illuminate\Http\Response
      */
-    public function store(Request $request)
+    public function store(TaskStoreRequest $request)
     {
-        $validator = Validator::make($request->all(), [
-            'name' => 'required|max:255',
-            'is_complete' => 'required',
-        ]);
-        if ($validator->fails()) {
-            return redirect()->route('task.create')
-            ->withInput()
-            ->withErrors($validator);
-      }
-      $task = new Task;
-      $task->name = $request->name;
-      $task->is_complete = $request->is_complete;
-      $task->save();
-      return redirect()->route('task.index')->with('success','Task Added successfully!');
+        $task = new Task;
+        $task->user_id = Auth::user()->id;
+        $task->name = $request->name;
+        $task->category_id = $request->category_id;
+        $task->is_complete = $request->is_complete;
```

Comment:
When a new todo is added, it should always be with `is_complete = 0`. Do not give the ability in the UI to create a new task which is completed.